service/javascript/assistants/syncassistant.js

```diff
@@ -1400,7 +1400,7 @@ var SyncAssistant = Class.create(Sync.SyncCommand, {
 					.setPersist(true)
 					.setReplace(true)
 					.setPower(true) //prevent narcolepsy
-					.setRequirements({ internet: true })
+					.setRequirements( undefined )
```

Comment:
shouldn't it be { undefined } ?

Reply:
I'm not 100% sure here either. We might even be able to drop the .setRequirements altogether. API documentation on OSE is not very clear.

Reply:
Ok, let's merge it like this as is for now